src/mono/browser/runtime/jiterpreter-support.ts

```diff
@@ -1,12 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-<<<<<<< HEAD
 import NativeAOT from "consts:nativeAOT";
```

Comment:
```suggestion
import NativeAOT from "consts:nativeAOT";

```
Future conflict prevention.